dependency-check-suppressions.xml

```diff
@@ -1,56 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
+	<!--Please add all the false positives under the below section-->
 	<suppress>
-		<notes>
-			Declared as False positive on library com.nimbusds:lang-tag #3594
-			https://github.com/jeremylong/DependencyCheck/issues/3594
-			CVE-2020-23171 Reference https://tools.hmcts.net/jira/browse/CCD-1871
+		<notes>False Positives
+			CVE-2020-23171 https://tools.hmcts.net/jira/browse/CCD-1871
+			CVE-2018-1258 https://tools.hmcts.net/jira/browse/CCD-765
+			CVE-2020-29242 https://tools.hmcts.net/jira/browse/CCD-92
+			CVE-2020-29243 https://tools.hmcts.net/jira/browse/CCD-70
+			CVE-2020-29244 https://tools.hmcts.net/jira/browse/CCD-104
+			CVE-2020-29245 https://tools.hmcts.net/jira/browse/CCD-287
+			CVE-2007-1651 https://tools.hmcts.net/jira/browse/CCD-125
+			CVE-2007-1652 https://tools.hmcts.net/jira/browse/CCD-115
+			CVE-2016-1000027 https://tools.hmcts.net/jira/browse/CCD-3255
 		</notes>
-		<packageUrl regex="true">^pkg:maven/com\.nimbusds/lang\-tag@.*$</packageUrl>
-		<cpe>cpe:/a:nim-lang:nim-lang</cpe>
 		<cve>CVE-2020-23171</cve>
-	</suppress>
-
-	<suppress>
-		<notes>We do not use: Spring Framework 5.0.5.RELEASE + Spring Security
-			(any version), see https://pivotal.io/security/cve-2018-1258
-			False positive confirmed.
-			Last control version: 5.5.1
-		</notes>
 		<cve>CVE-2018-1258</cve>
```

Comment:
shall we remove CVE-2018-1258 entirely?

Reply:
suppression of this is required

Reply:
refer https://github.com/hmcts/ccd-case-disposer/pull/74/files

Reply:
agreed as invalid